src/components/dashboard/news-item.vue

```diff
@@ -0,0 +1,51 @@
+<script>
+import TimeFormat from '~/mixins/time-format'
+export default {
+  name: 'news-item',
+  mixins: [TimeFormat],
+  components: {
+    Chips: () => import('~/components/common/chips.vue')
+  },
+  props: {
+    // news: () => {} // of { title, date, description, author }
+    news: Object
+  },
+  computed: {
+    tagType () {
+      let color
+      switch (this.news.type) {
+        case 'announcement':
+          color = 'primary'
+          break
+        case 'new feature':
+          color = 'grey-6'
+          break
+        default:
+          color = 'primary'
+          break
+      }
+      return [{
+        label: this.news.type.toUpperCase(),
```

Comment:
Avoid changing the text styling. Just capitalize the values where they are specified.

Maybe instead of `type` this is just a prop for `tag: Object` or maybe `tags: Array` if we wanted to allow multiple chips -- this object could include any props from the `chips` component such as the color, outline, etc. This allows us to be much more flexible on the news items and will make this component much simpler.